sdk/samples/iot/README.md

```diff
@@ -10,6 +10,7 @@
     - [IoT Hub SAS Telemetry Sample](#iot-hub-sas-telemetry-sample)
     - [IoT Hub Twin Sample](#iot-hub-twin-sample)
     - [IoT Plug and Play Sample](#iot-plug-and-play-sample)
+    - [IoT Plug and Play with Provisioning Sample](#iot-plug-and-play-with-provisioning-sample)
```

Comment:
Lines 55 and 59 below should include `paho_iot_provisioning_pnp_sample` (or whichever name is ultimately used, but I do agree with Eric I like this version).  Since it is both a DPS sample and a PnP sample.

Reply:
Responded to Eric above over naming but added it.